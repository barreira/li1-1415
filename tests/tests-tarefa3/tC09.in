aaaaa
aaaaa
aaaaa
AaaaA
0 0 N
LAD