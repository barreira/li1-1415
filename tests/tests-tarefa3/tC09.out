0 0
INCOMPLETO